ui/src/tag/TagSelector.tsx

```diff
@@ -37,11 +62,15 @@ export const TagSelector: React.FC<TagSelectorProps> = ({
     const container = React.useRef<null | HTMLDivElement>(null);
 
     const tagsResult = useQuery<Tags>(gqlTags.Tags);
+
     const suggestions = useSuggest(
```

Comment:
When typing a complete tag key in the text box, the tag should be added to the selectedTags when pressing the spacebar.